src/lib/utils/md.ts

```diff
@@ -279,19 +33,30 @@ const getPostSlugs = (dir: string, files: string[] = []) => {
       const fileExtension = extname(name)
 
       if (fileExtension === ".md") {
-        // If it is a .md file (allowed content page), push the path to the files array
-        for (const page of temporalAllowedPages) {
-          const fullPagePath = join(CURRENT_CONTENT_DIR, page)
-
-          if (name.includes(fullPagePath)) {
-            files.push(
-              toPosixPath(
+        if (process.env.ALLOWED_PAGES) {
+          const temporalAllowedPages = process.env.ALLOWED_PAGES.split(",")
+          // If it is a .md file (allowed content page), push the path to the files array
+          for (const page of temporalAllowedPages) {
+            const fullPagePath = join(CURRENT_CONTENT_DIR, page)
+
+            if (name.includes(fullPagePath)) {
+              files.push(
                 fullPagePath
                   .replace(CURRENT_CONTENT_DIR, "")
```

Comment:
these lines are being repeated on the 2 cases, could be moved to another aux function inside `md.ts`
```ts
// ...
.replace(CURRENT_CONTENT_DIR, "")
.replace("/index.md", "")
// For replacing forward slashes generated in windows file paths
.replace(/\\/g, "/")
```